### PR TITLE
Last Week in Pony - September 3, 2023

### PR DESCRIPTION
**Pull request by SeanTAllen**

### CODE REVIEW

File: content/blog/last-week-in-pony-090323.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
++++
+draft = false
+author = "seantallen"
+description = "Apparently Sean was getting paid by the commit this week."
+categories = [
+    "Last Week in Pony",
+]
+title = "Last Week in Pony September 3, 2023"
```
Comment (rhagenson) [inline content/blog/last-week-in-pony-090323.md:8]:
```suggestion
title = "Last Week in Pony - September 3, 2023"
```

File: content/blog/last-week-in-pony-090323.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+All done! Everything has been moved off of CirrusCI or ["decommissioned"](https://www.youtube.com/watch?v=yVDP5M0eTcM).
+
+### DockerHub to GitHub Container Registry Migration
+
+As we [mentioned last week](https://www.ponylang.io/blog/2023/08/last-week-in-pony-august-27-2023/#we-re-migrating-container-images-from-dockerhub-to-github-container-registry), we are migrating from DockerHub to GitHub Container Registry. On December 1st, we'll stop updating images in DockerHub. This past week we updated the following images:
+
+- `ponylang/changelog-bot-action`
+
+We've updated to post future releases while we migrate to both registries. It's unlikely we'll see another release before during the migration period so we uploaded tag 0.3.5 to GitHub Container Registry.
```
Comment (rhagenson) [inline content/blog/last-week-in-pony-090323.md:24]: I am not sure what this is meant to say. It is confusingly written. Do you mean that we have migrated to post to both and that it is unlikely there will be another release during the migration period?

File: content/blog/last-week-in-pony-090323.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+- `ponylang/changelog-tool`
+
+We've updated to post future releases while we migrate to both registries. It's unlikely we'll see another release before during the migration period so we uploaded tag 0.5.0 to GitHub Container Registry.
```
Comment (rhagenson) [inline content/blog/last-week-in-pony-090323.md:28]: Same text as above that I find confusing.

File: content/blog/last-week-in-pony-090323.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+- `ponylang/corral`
+- `ponylang/library-documentation-action`
+- `ponylang/library-documentation-action-v2`
+- `ponylang/ponyc`
+
+We've merged the PR to start uploading builds, both nightly and release, to GitHub Container Registry. By the time you read this, nightly images will be going in. On the next release, release images will be available via GitHub Container Registry as well.
```
Comment (rhagenson) [inline content/blog/last-week-in-pony-090323.md:35]: "going in" where? GHCR?

File: content/blog/last-week-in-pony-090323.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+- `ponylang/ponyup`
+- `ponylang/release-bot-action`
+- `ponylang/release-notes-bot-action`
+
+We've updated to post future releases while we migrate to both registries. It's unlikely we'll see another release before during the migration period so we uploaded tag 0.3.7 to GitHub Container Registry.
```
Comment (rhagenson) [inline content/blog/last-week-in-pony-090323.md:41]: Same confusing text.

File: content/blog/last-week-in-pony-090323.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+- `ponylang/release-notes-reminder-bot-action`
+
+We've updated to post future releases while we migrate to both registries. It's unlikely we'll see another release before during the migration period so we uploaded tag 0.1.1 to GitHub Container Registry.
```
Comment (rhagenson) [inline content/blog/last-week-in-pony-090323.md:45]: Same confusing text.

File: content/blog/last-week-in-pony-090323.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+That's everything that needs to move. Next up, on or after December 1st, we'll turn off pushing images to DockerHub. Please be sure to transition by then.
+
+### Pony Development Sync
+
+[Audio](https://sync-recordings.ponylang.io/r/2023_08_229.m4a) from the August 29th, 2023 sync is available.
```
Comment (rhagenson) [inline content/blog/last-week-in-pony-090323.md:51]:
```suggestion
[Audio](https://sync-recordings.ponylang.io/r/2023_08_29.m4a) from the August 29th, 2023 sync is available.
```

**Review by rhagenson — CHANGES_REQUESTED** [review]
A couple trivial changes that need to be made. As well, the stock text regarding the migration and unlikelihood of another release is confusing to me.

File: content/blog/last-week-in-pony-090323.md (hunk 3 above)
Comment (SeanTAllen) [reply content/blog/last-week-in-pony-090323.md:28]: update. please have a look.

File: content/blog/last-week-in-pony-090323.md (hunk 5 above)
Comment (SeanTAllen) [reply content/blog/last-week-in-pony-090323.md:41]: update. please have a look.

File: content/blog/last-week-in-pony-090323.md (hunk 6 above)
Comment (SeanTAllen) [reply content/blog/last-week-in-pony-090323.md:45]: update. please have a look.

File: content/blog/last-week-in-pony-090323.md (hunk 4 above)
Comment (SeanTAllen) [reply content/blog/last-week-in-pony-090323.md]: updated. please have another look

**Review by rhagenson — APPROVED** [review]
Looks excellent!

**Pull request merged**